src/lib/server-session.js

```diff
@@ -3,6 +3,10 @@ import { jwtDecode } from "jwt-decode";
 
 export function session() {
   const token = cookies().get("token")?.value;
-  const userData = jwtDecode(token);
-  return { token, userData };
+  if (token) {
+    const userData = jwtDecode(token);
+    return { token, userData };
+  } else {
+    return null;
+  }
 }
```

Comment:
Have you made sure this changes doesn't break other code?

Reply:
i've tested this and found no issue with the change. let me know if you have any other suggestions